include/mbedtls/ssl_internal.h

```diff
@@ -162,12 +177,14 @@
 #define MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC
 #endif
 
-#if defined(MBEDTLS_ARC4_C) || defined(MBEDTLS_CIPHER_NULL_CIPHER) ||   \
+#if defined(MBEDTLS_SSL_SOME_SUITES_USE_STREAM) ||
```

Comment:
The pre-processor complains about a missing `\` - can't entirely blame it.

Reply:
Fixed.